api/helpers/create_sequelize_schema/config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by TimH on 16-Mar-17.
+ */
+var config = module.exports = {};
+
+    //postgres database
+//config.hostname ="52.178.131.239";
+config.hostname ="127.0.0.1";
+config.port = "5432";
+config.database = "postgres";
+config.username = "postgres";
+config.password = "1234";
```

Comment:
Please make this file `json` so that you don't need to add exports and no one will ever dream of adding logic here.